packaging/firehol/firehol.functions

```diff
@@ -11,12 +11,12 @@ firehol_check_file() {
     sbin/Makefile.in)
       :
     ;;
-    configure.ac|sbin/commands.sed.in)
-      check_commands sbin/firehol.in || status=1
-      check_commands sbin/fireqos.in || status=1
-      check_commands sbin/link-balancer.in || status=1
-      check_commands sbin/update-ipsets.in || status=1
-      check_commands sbin/vnetbuild.in || status=1
+    configure.ac)
+      check_commands sbin/firehol.in.in || status=1
+      check_commands sbin/fireqos.in.in || status=1
+      check_commands sbin/link-balancer.in.in || status=1
+      check_commands sbin/update-ipsets.in.in || status=1
+      check_commands sbin/vnetbuild.in.in || status=1
```

Comment:
Well you are correct that having the firehol.config would reduce some checks.

Another check that takes place is looking for commands which have been included 'plain' e.g. just writing `modprobe` in the script without regard for the detected program. This is easily done (especially with more common commands) as these are scripts and will generally work 99.99% of the time, only to fail in weird ways unless we pick it up somewhere.